modules/oadp-installing-dpa.adoc

```diff
@@ -256,6 +270,53 @@ spec:
 <.> Specify the `csi` default plug-in if you use CSI snapshots to back up PVs. The `csi` plug-in uses the link:https://{velero-domain}/docs/main/csi/[Velero CSI beta snapshot APIs]. You do not need to configure a snapshot location.
 <.> The `openshift` plug-in is mandatory.
 <.> Set to `false` if you want to disable the Restic installation. Restic deploys a daemon set, which means that each worker node has `Restic` pods running. You configure Restic for backups by adding `spec.defaultVolumesToRestic: true` to the `Backup` CR.
+<.> Specify the node selector to be supplied to Restic podSpec
+<.> Specify the backup provider.
+<.> If you use a default plug-in for the backup provider, you must specify the correct default name for the `Secret`, for example, `cloud-credentials-gcp`. If you specify a custom name, the custom name is used for the backup location. If you do not specify a `Secret` name, the default name is used.
+<.> Specify a bucket as the backup storage location. If the bucket is not a dedicated bucket for Velero backups, you must specify a prefix.
+<.> Specify a prefix for Velero backups, for example, `velero`, if the bucket is used for multiple purposes.
+endif::[]
+ifdef::virt-installing-configuring-oadp[]
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: <dpa_sample>
+  namespace: openshift-adp
+spec:
+  configuration:
+    velero:
+      defaultPlugins:
+        - kubevirt <.>
+        - gcp <.>
+        - csi <.>
+        - openshift <.>
+    featureFlags:
```

Comment:
please remove lines 296-297, that's not necessary. Once adding csi to the list of defaultPlugins, this flag is added to the velero's container args.